Modix.Bot/Modules/AuthorizationModule.cs

```diff
@@ -17,85 +18,84 @@ namespace Modix.Modules
-                    : await AuthorizationService.GetGuildUserClaimsAsync(guildUser);
+            // If a user was provided, get their claims, otherwise get the caller's claims.
+            var claims = user is null
+                ? _authorizationService.CurrentClaims
+                : await _authorizationService.GetGuildUserClaimsAsync(user);
 
-            await ReplyWithClaimsAsync(claims);
+            await ReplyWithClaimsAsync(claims.ToList());
         }
 
         [Command("claims")]
         [Summary("Lists the currently assigned claims for the given role.")]
         public async Task ClaimsAsync(
-            [Summary("The role for which to list claims.")]
-                IRole guildRole)
+
+            [Summary("Role for which to list claims of.")]
+            IRole role)
+
         {
-            var claims = await AuthorizationService.GetGuildRoleClaimsAsync(guildRole);
+            var claims = await _authorizationService.GetGuildRoleClaimsAsync(role);
             await ReplyWithClaimsAsync(claims);
         }
 
-        private async Task ReplyWithClaimsAsync(IEnumerable<AuthorizationClaim> claims)
+        [Command("claims add")]
+        [Summary("Adds a claim to the given role")]
+        public Task AddClaim(
+
+            [Summary("Claim to be added")]
+            AuthorizationClaim claim,
+
+            [Summary("Access of a claim, whether granted or denied")]
+            ClaimMappingType type,
+
+            [Summary("Role which to add the claim to")]
+            IRole role)
+
         {
-            await ReplyAsync(claims.Any()
-                 ? Format.Code(string.Join("\r\n", claims.Select(x => x.ToString())))
-                 : "No claims assigned");
+            return _authorizationService.AddClaimMappingAsync(role, type, claim);
         }
 
-        [Command("claims add")]
-        [Summary("Adds a claim mapping to a given role")]
-        public Task AddClaimMapping(
-            [Summary("The claim to be added.")]
-                AuthorizationClaim claim,
-            [Summary("The type of claim mapping, e.g. granted or denied.")]
-                ClaimMappingType type,
-            [Summary("The role to which the claim is to be added.")]
-                IRole role)
-            => AuthorizationService.AddClaimMappingAsync(role, type, claim);
+        [Command("claims remove")]
+        [Summary("Removes a claim from the given role")]
+        public Task RemoveClaim(
 
-        [Command("claims add")]
-        [Summary("Adds a claim mapping to a given user")]
-        public Task AddClaimMapping(
-            [Summary("The claim to be added.")]
-                AuthorizationClaim claim,
-            [Summary("The type of claim mapping, e.g. granted or denied.")]
-                ClaimMappingType type,
-            [Summary("The user to which the claim is to be added.")]
-                IGuildUser user)
-            => AuthorizationService.AddClaimMappingAsync(user, type, claim);
```

Comment:
Why are we dropping this command?

Reply:
Oversight, will add.

Reply:
Looked again, it exists on line 54.

Reply:
...not in the version on GitHub. There's a command to add a claim to a role, but not a user.